lib/controller.js

```diff
@@ -92,7 +95,7 @@ Tessel.list = function(opts) {
 
 Tessel.get = function(opts) {
   return new Promise(function(resolve, reject) {
-    logs.info('Looking for your Tessel...');
+    logs.info('Searching nearby LAN (netmask) and USB for Tessels...');
```

Comment:
@johnnyman727 with the change of possible multiple `Looking for your Tessels ...` but being earnest: I think it is more useful to get an info about where and what is searched... 
The old version suggests the CLI is created for a single Tessel - and looking to the code I feel like this had bin the initial purpose ?!